### ISSUE TITLE
verify that tendrl-ansible is aligned with upstream documentation for particular release

### ISSUE THREAD

**Issue by mbukatov** [Tendrl/tendrl-ansible#9]
We need to verify that tendrl-ansible matches the instructions in upstream installation documentation exactly.

## Reasoning

While tendrl-ansible is an official way to install Tendrl, it's not a mandatory installation method and all steps automated in ansible roles and playbooks there should be documented in upstream installation documentation as well.

**Comment by mbukatov** [comment]
All my questions on tendrl-devel list has been answered and most important missing pieces added into the docs.

This means that I need to review the answers and suggest moving some explanations provided to me by tendrl devel team into the docs.

**Comment by mbukatov** [comment]
Blocked by https://github.com/Tendrl/documentation/issues/85

**Comment by mbukatov** [comment]
I'm closing it now as we are in the middle of dev phase again, but I should create a new issue for checking the next release, so that the issue refers to the particular release version properly.
